tools/runners/run-test-suite-test262.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+import sys
+import os
+import subprocess
+import shutil
+
+def get_platform_cmd_prefix():
+    if sys.platform == 'win32':
+        return ['cmd', '/S', '/C']
+    return ['python2']  # The official test262.py isn't python3 compatible, but has python shebang.
+
+
+def run_test262_tests(runtime, engine, path_to_test262):
+    if not os.path.isdir(os.path.join(path_to_test262, '.git')):
+        print (path_to_test262)
+        proc = subprocess.Popen(['git', 'clone', 'https://github.com/tc39/test262.git',
+                                 '-b', 'es5-tests', path_to_test262])
+        proc.wait()
+        if proc.returncode:
+            print('Cloning test262 repository failed.')
+            return proc.returncode
+
+    path = os.path.join(path_to_test262, 'test/suite/bestPractice')
```

Comment:
If we are already using `os.path.join` then we should use it all the way:
`os.path.join(path_to_test262, 'test', 'suite', 'bestPractice')`

In addition the `path` variable name does not say anything. It should be a bit more descriptive.

Reply:
Fixed.